public/app/features/alerting/unified/components/rules/central-state-history/EventListSceneObject.tsx

```diff
@@ -166,22 +201,24 @@ function EventRow({ record, logRecords }: EventRowProps) {
 interface AlertRuleNameProps {
   labels: Record<string, string>;
   ruleUID?: string;
+  addFilterByName: (alertRuleName: string) => void;
 }
-function AlertRuleName({ labels, ruleUID }: AlertRuleNameProps) {
+function AlertRuleName({ labels, ruleUID, addFilterByName }: AlertRuleNameProps) {
   const styles = useStyles2(getStyles);
+  const { pathname, search } = useLocation();
+  const returnTo = encodeURIComponent(`${pathname}${search}`);
```

Comment:
nit: can we use `createUrl()` so this also works with mounting Grafana on a sub path?

Reply:
(Good catch, I missed that in my previous suggestion, thanks!)